chainerrl/replay_buffer.py

```diff
@@ -153,6 +153,9 @@ def save(self, filename):
     def load(self, filename):
         with open(filename, 'rb') as f:
             self.memory = pickle.load(f)
+        if not isinstance(self.memory, RandomAccessQueue):
```

Comment:
`PrioritizedReplayBuffer.load` doesn't work well, because the class extends `ReplayBuffer` and its `memory` is not `RandomAccessQueue`.